ambry-cloud/src/main/java/com.github.ambry.cloud/azure/AzureCloudDestination.java

```diff
@@ -177,21 +172,12 @@ void testStorageConnectivity() {
    * Test connectivity to Azure CosmosDB
    */
   void testCosmosConnectivity() {
```

Comment:
Do we need this wrapper? Call `cosmosDataAccessor.testConnectivity()` directly?

Reply:
It's because the unit test calls this method, and I'd rather not expose the new class to the test.

Reply:
Fixed.